nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/test/java/org/apache/nifi/processors/iceberg/TestPutIcebergWithHiveCatalog.java

```diff
@@ -256,10 +256,13 @@ public void onTriggerUnPartitioned(String fileFormat) throws Exception {
         runner = TestRunners.newTestRunner(processor);
         initRecordReader();
         initCatalog(PartitionSpec.unpartitioned(), fileFormat);
-        runner.setProperty(PutIceberg.CATALOG_NAMESPACE, CATALOG_NAME);
-        runner.setProperty(PutIceberg.TABLE_NAME, TABLE_NAME);
+        runner.setProperty(PutIceberg.CATALOG_NAMESPACE, "${catalog.name}");
+        runner.setProperty(PutIceberg.TABLE_NAME, "${table.name}");
```

Comment:
Could you add the `maximumFileSize` property to the test, it also got support for `FLOWFILE_ATTRIBUTES` so would be nice to cover that too.